docs/src/design-system/charts/Charts.stories.tsx

```diff
@@ -0,0 +1,14 @@
+import { ComponentMeta, ComponentStory } from '@storybook/react';
```

Comment:
Also, there appears to be inconsistencies with the x-axis labels when compared with the designs.  There is a gap between the hour and minutes that aren't in the designs, 9 has a 0 before it that the designs don't have, and labels on the hour have minutes included when they should be removed according to the designs.  These are outside the scope of this ticket though so it might be better to create a ticket to address these.

Current:
![image](https://user-images.githubusercontent.com/106989342/202733713-d5a0742e-d701-4abd-bb8c-31e26aec6d40.png)

Figma Design:
![image](https://user-images.githubusercontent.com/106989342/202734760-43e95c0e-0586-445f-9ede-9b026e6e5f7a.png)

Reply:
Let me confirm this with Nate:)